project_start/express-api/routes/post.js

```diff
@@ -4,19 +4,16 @@ var request = require("request");
 
 const Parse = require("parse/node");
 const { default: axios } = require("axios");
-// Parse.initialize(
-//   "z81Jsr6Tc1lcHyxZK7a5psWRFOBuOs2e0nxXudMj",
-//   "JTrwOsEpJabYLzZVqKuG07FD5Lxwm2SzhM5EUVt5"
-// );
 
 Parse.initialize(
-  "YmpmeHLzpiEt1IiupexyPzd9vCgETDvaeW2rWh0U",
-  "8xOR0nDLQMijBppInKvJFsLXcDDfl7RwQ1d2QnNS"
+  "8mJaCOPGxTw5RVUHZ8Dfqx8oaZ5H0N4gTtfeIkrE",
```

Comment:
What are these keys?

Reply:
These are the "Application ID" and "JavaScript key" (both API keys from Parse).